cubeviz/layout.py

```diff
@@ -244,7 +250,16 @@ def _dict_to_menu(self, menu_dict):
         return menu_widget
 
     def _open_dialog(self, name, widget):
-        get_text(name, "What's your name?")
+        if name == "Filter":
+            ex = smoothing_gui.SelectSmoothing(
+                self._data, 
+                self.session.data_collection,
+                parent=self
+                ) 
+    def add_smoothed_cube_name(self, name):
```

Comment:
does this function need to be in this PR (given it says "smoothed")?  If not, let's remove for now.